problemsets-mat223/modules/module6-exercises.tex

```diff
@@ -1,17 +1,93 @@
 \begin{exercises}
+	% Topics
+	% formal + intuitive definition of subspace
+	% relationship between subspace and spans
+	% prove a set is a subspace
+	% basis + dimension of a subspace
 	\begin{problist}
-		\prob  Example problem with parts
+
+		\prob For each of the following sets, prove whether or not it is a subpace.
+		\begin{enumerate}
+			\item Let $\mathcal T \subseteq \R^2$ be the complete solution to $3x-y=0$.
+			\item Let $\mathcal U \subseteq \R^2$ be the complete solution to $\frac{1}{2}x-6y=0$.
+			\item Let $\mathcal V \subseteq \R^2$ be the complete solution to $x-5y-1=0$.
+			\item Let $\mathcal W \subseteq \R^3$ be the complete solution to $x+2y-3z=0$.
+			\item Let $\mathcal X \subseteq \R^3$ be the complete solution to
+			$5x-\pi y + (\ln 2)z=0$.
+			\item Let $n \in \N$. Let $a_1,...,a_n \in \R$. Let $\mathcal Q \subseteq \R^n$ be the complete solution to $a_1x_1+a_2x_2+...+a_{n-1}x_{n-1}+a_nx_n=0$.
```

Comment:
-> Let $\mathcal Q \subseteq \R^n$ be the complete solution to $a_1x_1+a_2x_2+...+a_{n-1}x_{n-1}+a_nx_n=0$ where $a_1,...,a_n \in \R$.

It's tradition to write variables in front of vectors...